run.py

```diff
@@ -270,16 +280,115 @@ def main(gtk_context):
 
             # This is what it is all about
             exec_command(
-                command, environ=environ, dry_run=dry_run, shell=True, cont_output=True,
+                command,
+                environ=environ,
+                dry_run=dry_run,
+                shell=True,
+                cont_output=True,
             )
 
+            # Harvest first level jsons into group level analysis
+            if hierarchy["run_level"] == "project":
+                command = generate_command(
+                    config,
+                    gtk_context.work_dir,
+                    output_analysis_id_dir,
+                    log,
+                    errors,
+                    warnings,
+                    analysis_level="group",
+                )
+
+                command_name = make_file_name_safe(command[0])
+                try:
+                    exec_command(
+                        command,
+                        environ=environ,
+                        dry_run=dry_run,
+                        shell=True,
+                        cont_output=True,
+                    )
+                except Exception as e:
+                    # Bare, extra exception from mriqc/cli/run.py line 113
+                    print(e)
+
+                # Copy the resulting tsv summaries to the enclosing output directory
+                # where the other, zipped output will live.
+                tsvs = glob.glob(os.path.join(output_analysis_id_dir, "*tsv"))
+                for tsv in tsvs:
+                    name_no_tsv = os.path.splitext(os.path.basename(tsv))[0]
+                    dest_tsv = os.path.join(
+                        context.output_dir,
+                        name_no_tsv + "_" + context.destination["id"] + ".tsv",
+                    )
+                    shutil.move(tsv, dest_tsv)
```

Comment:
This should probably be a copy instead of move so that they are also available in the "complete output" zip archive.

Reply:
Makes sense! Thanks.